public/Electron/ipc-handler.js

```diff
@@ -0,0 +1,67 @@
+const { ipcMain, BrowserWindow, BrowserView, dialog } = require("electron");
+const { exec } = require("child_process");
+const path = require("path");
+const waitOn = require("wait-on");
+const { fileInfo, portNumber, handleErrorMessage } = require("./utils");
+
+const registerIpcHandlers = () => {
+  ipcMain.handle("get-path", async () => {
+    try {
+      const { canceled, filePaths } = await dialog.showOpenDialog({
+        properties: ["openDirectory"],
+      });
+
+      if (!canceled && filePaths.length > 0) {
+        [fileInfo.filePath] = filePaths;
+
+        BrowserWindow.getFocusedWindow().webContents.send(
```

Comment:
함수내에서 자주 사용하는 부분은 변수처리할 수 있을 것 같아요
```suggestion
        const win = BrowserWindow.getFocusedWindow();
        win.webContents.send(
```